ui/app/AppLayouts/Wallet/popups/buy/BuyCryptoModalAdaptor.qml

```diff
@@ -0,0 +1,146 @@
+import QtQml 2.15
+import SortFilterProxyModel 0.2
+
+import StatusQ 0.1
+import StatusQ.Core.Utils 0.1
+
+import utils 1.0
+
+import shared.stores 1.0
+import AppLayouts.Wallet 1.0
+import AppLayouts.Wallet.stores 1.0 as WalletStore
+
+QObject {
+    id: root
+
+    required property WalletStore.BuyCryptoStore buyCryptoStore
+    required property BuyCryptoParamsForm buyCryptoFormData
+    required property var walletAccountsModel
+    required property var networksModel
+    required property bool areTestNetworksEnabled
+    required property var groupedAccountAssetsModel
+    required property var plainTokensBySymbolModel
+    required property string currentCurrency
```

Comment:
This seems completely unused?

Reply:
it is needed here https://github.com/status-im/status-desktop/pull/16002/files#diff-f362becfa7f2d874d446e2e9372982cc0854d059ddb2e0bb076c8df62e3e51a9R118:~:text=currentCurrency%3A%20root.buyCryptoAdaptor.currentCurrency

Reply:
Ah I see!